src/lexical-elements.rst

```diff
@@ -976,27 +970,22 @@ Character Literals
 
    CharacterContent ::=
        AsciiEscape
+     | UnicodeEscape
```

Comment:
Put this line after `CharacterLiteralCharacter` as it is useful to keep the alternatives alphabetized.